app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt

```diff
@@ -335,4 +335,21 @@ class TripsViewModelTest {
     tripsViewModel.setInitialUiState()
     assert(tripsViewModel.uiState.value is UiState.Initial)
   }
+
+  @Test
+  fun testGetFeed() {
+    tripsViewModel.getFeed("userId")
+    verify(tripsRepository).getFeed(any(), any(), any())
+  }
+
+  @Test
```

Comment:
Can you please also add tests for these 2 methods in TripsViewModelTest?

Reply:
I do not really get you comment the getFeed function I added in the TripViewModel is tested in this PR. What 2 methods are you referring to?